### PR TITLE
Feature SFSnackBar component

### PR DESCRIPTION
**Pull request by Yopo-Nicolas**

### CODE REVIEW

**Review by pdcastano — CHANGES_REQUESTED** [review]

File: src/Components/SFSnackbar/SFSnackbar.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+const messageStyles = makeStyles((theme: Theme) => ({
+  mesageStyle: {
+    color: theme.palette.type === 'light' ? SFGrey[50] : SFGrey[900],
+    margin: 0
+  }
+}));
+
+export interface SFSnackBarProps extends Omit<SnackbarProps, 'message'> {
+  buttonText: string;
+  message: string;
```
[the remark this answers is not in the document]
Comment (Yopo-Nicolas) [reply src/Components/SFSnackbar/SFSnackbar.tsx]: Changed

File: src/Components/SFSnackbar/SFSnackbar.tsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  onClick?: () => void;
+}
+
+export const SFSnackBar = ({
+  buttonText,
+  open = true,
```
[the remark this answers is not in the document]
Comment (Yopo-Nicolas) [reply src/Components/SFSnackbar/SFSnackbar.tsx]: Done

**Review by pdcastano — CHANGES_REQUESTED** [review]
* background-color and box-shadow props don't match with Figma (DM/NM)

**Review by pdcastano — CHANGES_REQUESTED** [review]

File: src/Components/SFSnackbar/SFSnackbar.tsx [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+  onClick?: () => void;
+}
+
+export const SFSnackBar = ({
+  buttonText,
+  open,
+  message,
+  anchorOrigin = { vertical: 'top', horizontal: 'center' },
+  onClick
+}: SFSnackBarProps): React.ReactElement<SFSnackBarProps> => {
```
[the remark this answers is not in the document]
Comment (Yopo-Nicolas) [reply src/Components/SFSnackbar/SFSnackbar.tsx]: done

**Review by pdcastano — COMMENTED** [review]
Hover and Click styles in the action button are wrong in night mode. (Check also colors in day mode)

**Comment by pdcastano** [comment]
LGTM

**Pull request merged**